cypress/fixtures/circuits/params.json

```diff
@@ -0,0 +1,113 @@
+{
+  "phase":"2 * \\pi / 3",
+  "commands":[
+    {"op":{"type":"U3","params":["1.0","-0.56666666","Abs(0.5)"]},"args":[["q",[0]]]},
+    {"op":{"type":"Sdg"},"args":[["q",[1]]]},
+    {"op":{"type":"U3","params":["sinh(theta / 2) / acosh(phi**2)","<hrkhrk","I'm evil>:)"]},"args":[["q",[2]]]},
+    {"op":{"type":"Rx", "params":["</div>"]},"args":[["q",[3]]]},
+    {
+      "op": {
+        "conditional": {
+          "op": {
+            "conditional": {
+              "op": {"type": "Phase",  "params": ["0.1345678"]},
+              "value": 0,
+              "width": 1
+            },
+            "type": "Conditional"
+          },
+          "value": 1,
+          "width": 1,
+          "args": [["c", [1]]]
+        },
+        "type": "Conditional"
+      },
+      "args": [["c", [0]]]
+    },
+    {"op":{"type":"Rz", "params":["ceiling(sqrt(pi/2)) + floor(cbrt(2345))"]},"args":[["q",[0]]]},
+    {"op": {"type": "Rx", "params": ["0.25 * pi"]}, "args": [["q", [2]]]},
+    {"op": {"type": "Ry", "params": ["atan2(x,y)"]}, "args": [["q", [3]]]},
+    {"op":{"type":"CircBox","box":{"id":"621ee04f-bc1f-4ae3-bbf5-0db8385e9e01","type":"CircBox","circuit":{
+      "name": "A $$\\mathbb{COOL}$$ circuit.",
+      "phase":"0.0",
+      "qubits":[["q",[0]],["q",[1]]],
+      "bits":[],
+      "commands":[
+        {"op":{"type":"SWAP"},"args":[["q",[1]],["q",[0]]]},
+        {"op":{"type":"Rx","n_qb":1,"params":["cy", "alpha^2 + beta^3"]},"args":[["q",[1]]]},
+        {"op": {"type": "Phase", "params": ["0.5"]}, "args": null},
+        {"op": {"type": "Phase", "params": ["0.5"]}, "args": null},
+        {"op":{"type":"CZ"},"args":[["q",[0]],["q",[1]]]},
+        {"op":{"type":"CX"},"args":[["q",[1]],["q",[0]]]}
+      ],
+      "implicit_permutation":[[["q",[0]],["q",[0]]],[["q",[1]],["q",[1]]],[["q",[2]],["q",[2]]],[["q",[3]],["q",[3]]]]}}},
+      "args":[["q",[0]],["q",[1]]]
+    },
+    {"op":{"type":"CircBox","box":{"id":"621ee04f-bc1f-4ae3-bbf5-0db8385e9e01","type":"CircBox","circuit":{
+      "name": "$$U_1\\,\\otimes\\,U_2$$",
```

Comment:
This box doesn't look quite right

Is this just because $U_1\\,\\otimes\\,U_2$ takes up too much space?

<img width="135" alt="Screenshot 2024-07-12 at 12 21 00" src="https://github.com/user-attachments/assets/572ccdb9-aa8e-4bdb-8242-1aaa42338743">

I understand that ensuring expressions are of suitable length is left up to the user

Reply:
Hmm that shouldn't happen.. (I checked in firefox and chrome and couldn't reproduce it - what version are you using again?)

Reply:
I'm using Firefox Version 128.0 on my M1 Mac

Reply:
This is fixed now. Thanks.